server/src/hardware/protocol/z21/clientkernel.hpp

```diff
@@ -62,6 +62,14 @@ class ClientKernel final : public Kernel
   private:
     const bool m_simulation;
     boost::asio::steady_timer m_keepAliveTimer;
+    BroadcastFlags m_broadcastFlags;
+
+    static constexpr BroadcastFlags RequiredBCFlags =
```

Comment:
Please rename it to `requiredBroadcastFlags`.
Variable names are lowerCamelCase.
I prefer to don't abbreviate, just to be clear.